tilecloud_chain/controller.py

```diff
@@ -279,46 +310,64 @@ def _fill_legend(
         assert gene.config_file
         config = gene.get_config(gene.config_file)
 
+    start = time.perf_counter()
+    legend_image_future = {}
+    with concurrent.futures.ThreadPoolExecutor(
+        max_workers=int(os.environ.get("TILECLOUD_CHAIN_CONCURENT_GET_LEGEND", "20"))
```

Comment:
please document in the readme the usage of `TILECLOUD_CHAIN_CONCURENT_GET_LEGEND`